github/authorizations.go

```diff
@@ -274,14 +274,18 @@ func (s *AuthorizationsService) Delete(ctx context.Context, id int64) (*Response
 // The returned Authorization.User field will be populated.
 //
 // GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#check-an-authorization
-func (s *AuthorizationsService) Check(ctx context.Context, clientID string, token string) (*Authorization, *Response, error) {
-	u := fmt.Sprintf("applications/%v/tokens/%v", clientID, token)
+func (s *AuthorizationsService) Check(ctx context.Context, clientID string, clientToken string, accessToken string) (*Authorization, *Response, error) {
+	u := fmt.Sprintf("applications/%v/token", clientID)
 
-	req, err := s.client.NewRequest("GET", u, nil)
+	req, err := s.client.NewRequest("POST", u, nil)
 	if err != nil {
 		return nil, nil, err
 	}
 
+	q := req.URL.Query()
+	q.Add("access_token", accessToken)
+	req.URL.RawQuery = q.Encode()
```

Comment:
I don't think these 3 lines are correct.

If I'm reading this page correctly: https://developer.github.com/v3/apps/oauth_applications/#check-a-token
it is saying that the access token is being passed in the body of the request, not as a query parameter.

This means that instead of sending here, line 280 above would have to pass a struct, something like this:

```
u := fmt.Sprintf("applications/%v/token", clientID)
reqBody := &struct{
  AccessToken string `json:"access_token"`
}{ AccessToken: accessToken }
req, err := s.client.NewRequest("POST", u, reqBody)
```